lms/djangoapps/grades/course_grade.py

```diff
@@ -156,6 +156,27 @@ def summary(self):
         grade_summary['grade'] = self.letter_grade
         return grade_summary
 
+    def _prep_course_for_grading(self):
+        """
+        Make sure any overrides to the grading policy are used.
+        This is most relevant for CCX courses.
+
+        Right now, we still access the grading policy from the course
+        object. Once we get the grading policy from the BlockStructure
+        this will no longer be needed - since BlockStructure correctly
+        retrieves/uses all field overrides.
```

Comment:
Thanks for this docstring. Whoever ends up working on this code down the line will appreciate it!